sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=GetDKAN_dkan
```

Comment:
Are we using sonar?

Reply:
OOps, was expirementing with it, meant to move out to other branch/PR